packages/admin/components/api/SchemaInstanceForm/Factories/GroupProperty/GroupProperty.tsx

```diff
@@ -18,13 +18,7 @@ export default function GroupProperty(props: Props) {
         ))}
       </FormGrid>
     </FieldsetSection>
-  ) : (
-    <>
-      {group.properties.map((prop, index) => (
-        <SchemaProperty field={prop} key={index} />
-      ))}
-    </>
-  );
+  ) : null;
```

Comment:
Let's still show the group, maybe with a default legend "Fields". Otherwise we might not realize that's why the fields aren't showing in the admin app